code/modules/client/preference_setup/loadout/loadout_xeno/machine.dm

```diff
@@ -166,4 +166,11 @@
 	description = "An identification card given to free IPC residents within the Republic of Biesel."
 	path = /obj/item/clothing/accessory/badge/passcard/synthetic
 	cost = 0
+	whitelisted = list(SPECIES_IPC, SPECIES_IPC_G1, SPECIES_IPC_G2, SPECIES_IPC_XION, SPECIES_IPC_ZENGHU, SPECIES_IPC_BISHOP, SPECIES_IPC_SHELL)
+/datum/gear/accessory/ipcmantle
+	display_name = "Burzsiean shoulder mantle"
```

Comment:
```suggestion
	display_name = "Burzsian shoulder mantle"
```

its not actually written anywhere official, but Burzsian seems to be the agreed way of spelling this